troop/models/participant.py

```diff
@@ -0,0 +1,130 @@
+from django.db import models
+from django.core.validators import MaxValueValidator
+from django.utils.formats import localize
+from django.utils.translation import gettext_lazy as _
+
+from .attendance import Attendance
+from .diet import Diet
+from .troop import Troop
+
+
+class Participant(models.Model):
+    """ Participant of the scout camp
+
+    Columns:
+        :troop: scout troop to which the participant belongs
+        :firstname: given name
+        :lastname: surname
+        :gender: gender
+        :birthday: date of birth
+        :email: email address
+        :nami: scouting membership number (nami)
+        :age_section: section to which the participant belongs
+        :is_leader: true if participant is a scout leader
+        :attendance: dates present on the camp site
+        :diet: special diet requirements like vegetarian
+        :medication: information about diseases and drugs
+        :comment: additional information
+        :deregistered_at: timestamp of deregistration (if unregistered)
+        :created_at: timestamp of creation of this record
+        :updated_at: timestamp of last update to this record
+    """
+
+    GENDER_MALE = "male"
+    GENDER_FEMALE = "female"
+    GENDER_DIVERSE = "diverse"
+
+    GENDER_CHOICES = (
+        (GENDER_MALE, _("male")),
+        (GENDER_FEMALE, _("female")),
+        (GENDER_DIVERSE, _("diverse")),
+    )
+
+    SECTION_BEAVER = "beaver"
+    SECTION_CUB = "cub"
+    SECTION_SCOUT = "scout"
+    SECTION_VENTURER = "venturer"
+    SECTION_ROVER = "rover"
+
+    SECTION_CHOICES = (
+        (None, _("no section")),
+        (SECTION_BEAVER, _("beaver")),
+        (SECTION_CUB, _("cub")),
+        (SECTION_SCOUT, _("scout")),
+        (SECTION_VENTURER, _("venturer")),
+        (SECTION_ROVER, _("rover")),
+    )
+
+    troop = models.ForeignKey(Troop, verbose_name=_("troop"), on_delete=models.CASCADE,)
+
+    firstname = models.CharField(_("firstname"), max_length=128,)
```

Comment:
Since this is a breaking change anyway, could you rename `firstname` to `first_name` and `lastname` to `last_name`?